config/sc-machine.ini

```diff
@@ -12,6 +12,9 @@ log_type = Console
 log_file = ../sc-memory.log
 log_level = Info
 
+result_structure_system_idtf = resultStructure
+result_structure_upload = true
```

Comment:
maybe duplicated is not a good idea? What if memory result_structure_system_idtf != builder  result_structure_system_idtf ? should stay parameters in builder section or different names for parameters in memory section